python/ml4ir/base/model/architectures/auto_dag_network.py

```diff
@@ -15,36 +15,7 @@
 from ml4ir.base.config.keys import FeatureTypeKey
 from ml4ir.base.features.feature_config import FeatureConfig
 from ml4ir.base.io.file_io import FileIO
-
-
-def get_layer_subclasses() -> Dict[str, Type[layers.Layer]]:
```

Comment:
I'm moving this to a common utils module because we will use it for both `AutoDagNetwork` and `DNN`.